src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -6,12 +6,24 @@
 			</h3>
 			<!-- First category row: Recommended loans section -->
 			<lending-category-section
+				title="Recommended for you"
+				subtitle="Loans handpicked for you based on your lending history"
 				:loans="recommendedLoans"
+				:per-step="2"
 				class="tw-mt-2"
-				@add-to-basket="trackRecommended"
+				@add-to-basket="trackCategory($event, 'recommended')"
 			/>
 
 			<quick-filters-section class="tw-mt-6" />
+
+			<!-- Second category row: Matched loans section -->
+			<lending-category-section
```

Comment:
Based on the slack convo, it sounds like this section should be hidden until matched worked as expected in FLSS?

Reply:
same above